python/openmldb_tool/diagnostic_tool/parser.py

```diff
@@ -78,3 +93,81 @@ def zk_conn_err(self):
         checker = StatusChecker(conn)
         assert checker._get_components(show=False), "Failed to connect to zk"
         self.result += "\nSuccessfully checked zk connection. It may be caused by `Too many connections` in zk server. Please check zk server log."
+
+
+class ProtoParser:
```

Comment:
remove it

Reply:
bak commit https://github.com/zhangziheng01233/OpenMLDB/commit/71e5b13a83b55c3521b5d02462969fea112c7758